.github/workflows/prepare-for-prod-ut-deploy.yml

```diff
@@ -0,0 +1,135 @@
+      img_tag: ${{ needs.generate-tag-names.outputs.tag_name_ut }}
+      dockerfile: Dockerfile-ut-func
+      load_target: development
+      push_target: production
+    secrets:
+      DOCKERHUB_PROD_TOKEN: ${{ secrets.DOCKERHUB_PROD_TOKEN }}
+
+  create-pull-request:
+    name: Update Helm Charts For Production and Create Pull Request
+    runs-on: ubuntu-latest
+    needs: [generate-tag-names, build-user-transformer-image]
+    env:
+      TAG_NAME: ${{ needs.generate-tag-names.outputs.tag_name }}
+      UT_TAG_NAME: ${{ needs.generate-tag-names.outputs.tag_name_ut }}
+      TF_IMAGE_REPOSITORY: rudderstack/rudder-transformer
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.5.2
+        with:
+          fetch-depth: 1
+
+      # In order to make a commit, we need to initialize a user.
+      # You may choose to write something less generic here if you want, it doesn't matter functionality wise.
+      - name: Initialize Mandatory Git Config
+        run: |
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "noreply@github.com"
+
+      - name: Print Docker Image Tags
+        run: |
+          echo "Transformer: $TAG_NAME"
+          echo "User Transformer: $UT_TAG_NAME"
+
+      - name: Clone Devops Repo
+        run: |
+          git clone https://${{secrets.PAT}}@github.com/rudderlabs/rudder-devops.git
+
+      - name: Extract branch name
+        id: extract_branch_name
+        run: |
+          cd rudder-devops
+          branch_name=$(git rev-parse --abbrev-ref HEAD)
+          echo "branch_name=$branch_name"
+          echo "branch_name=$branch_name" >> $GITHUB_OUTPUT
+
+      - name: Update Helm Charts and Raise Pull Request
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+        run: |
+          cd rudder-devops
+          git checkout -b shared-user-transformer-$TAG_NAME
+
+          cd helm-charts/shared-services/per-az/environment/production
+
+          cd ../../config-be-rudder-transformer
+          yq eval -i ".config-be-user-transformer.image.tag=\"$UT_TAG_NAME\"" values.prod.yaml
+          yq eval -i ".config-be-user-transformer.image.repository=\"$TF_IMAGE_REPOSITORY\"" values.prod.yaml
+          git add values.prod.yaml
+
+          git commit -m "chore: upgrade shared user-transformers to $TAG_NAME"
+          git push -u origin shared-user-transformer-$TAG_NAME
+
+          hub pull-request -m "chore: upgrade shared user-transformers to $TAG_NAME"
+
+      - name: Update Helm Chart and Raise Pull Request For Hosted Transformer
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+        run: |
+          cd rudder-devops
+          git checkout -b hosted-user-transformer-$TAG_NAME ${{steps.extract_branch_name.outputs.branch_name}}
+
+          cd customer-objects/multi-tenant-us
+          yq eval -i ".spec.transformer.image.version=\"$TAG_NAME\"" hostedmtedmt.yaml
```

Comment:
Remove this once implemented following
https://github.com/rudderlabs/rudder-transformer/pull/2332#discussion_r1255617793